mappings/net/minecraft/menu/NameableMenuFactory.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_3908 net/minecraft/menu/NameableMenuFactory
+	METHOD method_5476 getDisplayName ()Lnet/minecraft/class_2561;
```

Comment:
Should be like `getTitle`

Reply:
I'm going to revert that change since it's the same obf/intermediary method as `Nameable.getDisplayName` which is used by entities like players.